README.md

```diff
@@ -1,5 +1,16 @@
-# docs-community
+# Python Docs WG
 
 [![Documentation Status](https://readthedocs.org/projects/docs-community/badge/?version=latest)](https://docs-community.readthedocs.io/en/latest/?badge=latest)
- 
-Community management for documentation contributors and the Docs Workgroup
+
+The PSF Documentation Working Group (Docs WG) was chartered by the PSF to help bootstrap a larger Python documentation team and community.
```

Comment:
Does it make sense to include "Python" in the full name, since it is specifically for the documentation for the Python language (though I suppose that is what the P in PSF stands for)?

Repetition of PSF without defining what it stands for, and since this repo is under the Python org, not the PSF one, it would make sense to link the [PSF](https://www.python.org/psf/) page.

```suggestion
The PSF Documentation Working Group (Docs WG) was chartered by the [Python Software Foundation](https://www.python.org/psf/) to help bootstrap a larger documentation team and community.
```